### PR TITLE
Fix documentation for new float_precision on read_csv

### PR DESCRIPTION
**Pull request by Dr-Irv**
- [x] closes https://github.com/pandas-dev/pandas/pull/36228#discussion_r487587534
- [ ] tests added / passed
     N/A
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry
     N/A

A followup to PR #36228 to get the documentation string right for version changed. (was in wrong place, missing blank line)

### CODE REVIEW

File: pandas/io/parsers.py [hunk 1]
```diff
@@ -2281,9 +2283,10 @@ def TextParser(*args, **kwds):
         can be inferred, there often will be a large parsing speed-up.
     float_precision : str, optional
         Specifies which converter the C engine should use for floating-point
-        values. The options are None for the ordinary converter,
-        'high' for the high-precision converter, and 'round_trip' for the
-        round-trip converter.
+        values. The options are `None` or `high` for the ordinary converter,
+        `legacy` for the original lower precision pandas converter, and
+        `round_trip` for the round-trip converter.
```
Comment (jbrockmendel) [inline pandas/io/parsers.py:2288]: should high/legacy/round_trip have ""s to denote they are strings?

Comment (Dr-Irv) [reply pandas/io/parsers.py:2298]: Good point.  The options didn't have quotes before, so now with my latest commit, they do.

**Comment by Dr-Irv** [comment]
@jreback @jbrockmendel this is all green - just a doc change so it can be merged

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @Dr-Irv

**Pull request merged**